services/ui-src/src/components/reports/ReportPageWrapper.tsx

```diff
@@ -0,0 +1,59 @@
+// components
+import { Flex } from "@chakra-ui/react";
+import { PageTemplate, Sidebar, StandardReportPage } from "components";
+// utils
+
+import React from "react";
+import { mockStandardReportPageJson } from "utils/testing/mockForm";
+
+export const ReportPageWrapper = () => {
+  return (
+    <PageTemplate type="standard">
```

Comment:
I think we can swap this type to report, it'll make the display of the page much nicer. To do that you'll need to change `type?: "standard"` on line 25 of PageTemplate.tsx to `type?: "standard" | "report";`